src/main.py

```
@@ -13,9 +15,23 @@
 from commands import *
 
 
+
 intents = discord.Intents.default()
 bot = discord.Bot(intents=intents)
 
+
+from webhook import *
+# Run Flask in a separate thread
+def run_flask(bot):
+    app.bot = bot  # Attach the bot instance to the Flask app
```

Ruff (0.8.2)

`app` may be undefined, or defined from star imports

(F405)